add notes about `skip_credentials_validation` (#7823)
terraform-providers/terraform-provider-azurerm: website/docs/index.html.markdown

> * `skip_credentials_validation` - (Optional) Should the AzureRM Provider skip verifying the credentials being used are valid? This can also be sourced from the `ARM_SKIP_CREDENTIALS_VALIDATION` Environment Variable. Defaults to `false`.
> 
> ~> **Note:** if `skip_credentials_validation` is false, AzureRM Provider will send request to list all available providers.
> 

Reviewer: ```suggestion
* `skip_credentials_validation` - (Optional) Should the AzureRM Provider skip verifying the credentials being used are valid? If `false` the provider will send a request to list all available Resource Providers. This can also be sourced from the `ARM_SKIP_CREDENTIALS_VALIDATION` Environment Variable. Defaults to `false`.

```
